resources/index.html

```diff
@@ -24,7 +24,7 @@
   <h1>Sketch</h1>
   <p>A Sketch template, as well as a Craft Library are available via OneDrive.</p>
 
-  <a class="ui button ds-btn-med" href="https://raxglobal-my.sharepoint.com/personal/jacki_bauer_rackspace_com/_layouts/15/guestaccess.aspx?guestaccesstoken=BClyS4a2YRNTmFP1IQYmidFZAyI9/CG5Rzfbs3dt7/E=&folderid=2_106f98f35849f4a0d807eef923f3d8429&rev=1" target="_blank">View Sketch Resources</a>
+  <a class="ui button ds-btn-med" href="https://github.com/rackerlabs/helix" target="_blank">View Sketch Resources</a>
```

Comment:
I believe that since the Helix repo is currently private, the user won't be able to see the GH page unless they have been added to repo.  Let's consider either uploading a zip to CDN or providing the link to the file on SharePoint for now.